Guide/src/dev_guide/contrib/code.md

```diff
@@ -152,7 +152,7 @@ e.g:
 ```rust
 // x86_64-unknown-linux-musl targets have a different type defn for
 // `libc::cmsghdr`, hence why these lints are being suppressed.
-#[allow(clippy::needless_update, clippy::useless_conversion)]
+#[expect(clippy::needless_update, clippy::useless_conversion)]
```

Comment:
This example probably won't compile because it's different on different targets.